source/__tests__/integration/create-profile.test.js

```diff
@@ -0,0 +1,51 @@
+describe('Validating the profile metadata', () => {
+
+  beforeAll(async () => {
+    await page.goto('http://localhost:8000/#/profile/create/')
+  })
+
+  describe('with the required profile metadata fields', () => {
+
+    afterEach(async () => {
+      page.$eval('form[name="profileForm"]', e => e.reset())
+    })
+
+    it.each`
```

Comment:
I like this construction of this test using a table instead of individual tests. This matrix makes easier to understand what condition you're testing in the form.